.travis.yml

```diff
@@ -47,17 +47,11 @@ matrix:
       dist: bionic
       env: USE_PSALM=0
            CHECK_MBSTRING=0
-    - php: "master"
```

Comment:
Why is this removed?

Reply:
What's the difference between `master` and `nightly`?

Reply:
At one point, one was "latest nightly build of the new version" and the other was supposed to be "built from the master branch" (which in the near future should be called the `main` branch). But it looks like only `nightly` even exists right now.